integreat_cms/cms/utils/content_utils.py

```diff
@@ -162,3 +163,33 @@ def fix_alt_texts(content: HtmlElement) -> None:
                 image.attrib["alt"] = media_file.alt_text
         else:
             logger.warning("Empty img tag was found.")
+
+
+def hide_anchor_tag_around_image(content: HtmlElement) -> None:
+    """
+    This function checks whether an image tag wrapped by an anchor tag has an empty alt tag, if so it hides anchor tag from screen-reader and tab-key
+
+    :param content: the content which has an anchor tag wrapped around an img tag
+    """
+
+    for anchor in content.iter("a"):
+        href = anchor.attrib.get("href")
+        # Check if the href attribute ends with the specified image extensions
+        if href.endswith(("png", "jpeg", "svg")):
```

Comment:
Actually, is there any problem if we just remove the check for file type? To me it seems like this would be generally useful for all `<img>` tags.

Reply:
I think apart from image files, we do not use any anchor tag, so ýou are right :+1:

Reply:
Good point. In that case, wouldn't it make sense to just remove the for loop through anchors and the href check for filetype and instead just loop through all images?

Reply:
@david-venhoff but we need to find the href endings with those filetypes so that we can hide them from screen readers (specifically anchor tags) They are not needed when the image has no alt-text :thinking:

Reply:
Ah, I get it now :see_no_evil: 

Though I still would prefer if we did not perform a check for file type. Instead maybe we could test if the anchor tag has no text and only <img> children?

Reply:
sounds good :+1: